### ISSUE TITLE
URLs with slash at end create double slashes

### ISSUE THREAD

**Issue by krlmlr** [r-lib/pkgdown#1464]
I think we should be able to use URLs from the `URL` field in `DESCRIPTION` which now often must end with a slash due to `R CMD check's` witch hunt on redirects.

Now: `DESCRIPTION` must have URLs that end with a slash, `_pkgdown.yml` can't have a `url` field that ends with a slash.

These double URLs propagate to `downlit::autolink_url()` which make the result look a little ugly.

``` r
test <- usethis::create_package("test")
#> ✓ Creating 'test/'
#> ✓ Setting active project to '/tmp/Rtmpk3bUiG/reprex6ed615788f7f/test'
#> ✓ Creating 'R/'
#> ✓ Writing 'DESCRIPTION'
#> Package: test
#> Title: What the Package Does (One Line, Title Case)
#> Version: 0.0.0.9000
#> Authors@R (parsed):
#>     * First Last <first.last@example.com> [aut, cre] (YOUR-ORCID-ID)
#> Description: What the package does (one paragraph).
#> License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
#>     license
#> Encoding: UTF-8
#> LazyData: true
#> Roxygen: list(markdown = TRUE)
#> RoxygenNote: 7.1.1.9000
#> ✓ Writing 'NAMESPACE'
#> ✓ Setting active project to '<no active project>'

pkg <- pkgdown::as_pkgdown(test, override = list(url = "https://bog.us/"))

pkgdown:::site_meta(pkg)
#> pandoc: 2.11.1.1
#> pkgdown: 1.6.1.9000
#> pkgdown_sha: ~
#> articles: {}
#> last_built: 2021-01-07T13:42Z
#> urls:
#>   reference: https://bog.us//reference
#>   article: https://bog.us//articles
```

<sup>Created on 2021-01-07 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jayhesselberth** [comment]
Related to #1439

**Comment by maelle** [comment]
This should be fixed by #1526 

``` r
td <- withr::local_tempdir()
#> Setting deferred event(s) on global environment.
#>   * Execute (and clear) with `withr::deferred_run()`.
#>   * Clear (without executing) with `withr::deferred_clear()`.
test <- usethis::create_package(td, open = TRUE)
#> ✓ Creating '/tmp/Rtmp87D8x3/file111115ff8586d/'
#> ✓ Setting active project to '/tmp/Rtmp87D8x3/file111115ff8586d'
#> ✓ Creating 'R/'
#> ✓ Writing 'DESCRIPTION'
#> Package: file111115ff8586d
#> Title: What the Package Does (One Line, Title Case)
#> Version: 0.0.0.9000
#> Authors@R (parsed):
#>     * First Last <first.last@example.com> [aut, cre] (YOUR-ORCID-ID)
#> Description: What the package does (one paragraph).
#> License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
#>     license
#> Encoding: UTF-8
#> LazyData: true
#> Roxygen: list(markdown = TRUE)
#> RoxygenNote: 7.1.1.9001
#> ✓ Writing 'NAMESPACE'
pkg <- pkgdown::as_pkgdown(test, override = list(url = "https://bog.us/"))
pkgdown:::site_meta(pkg)
#> pandoc: 2.11.2
#> pkgdown: 1.6.1.9000
#> pkgdown_sha: ~
#> articles: {}
#> last_built: 2021-02-23T16:02Z
#> urls:
#>   reference: https://bog.us/reference
#>   article: https://bog.us/articles
```

<sup>Created on 2021-02-23 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0.9001)</sup>
